src/tests/ftest/util/ior_utils.py

```diff
@@ -150,6 +152,7 @@ def set_daos_params(self, group, pool, cont_uuid=None, display=True,
             cont_uuid (str, optional): the container uuid. If not specified one
                 is generated. Defaults to None.
             display (bool, optional): print updated params. Defaults to True.
+            mpiio_oclass (int, optional): obj class when using mpiio api
```

Comment:
Depending on your opinion on the previous comment, this should be removed (+line 146)